src/components/Header/Menu.tsx

```diff
@@ -52,8 +54,6 @@ export function MenuLink({ name, link, className }: MenuItem) {
 export default function Menu(): ReactElement {
   const { appConfig, siteContent } = useMarketMetadata()
 
-  const { setIsAutomationEnabled } = useAutomation()
-
```

Comment:
question: Is this intentional? Seems unrelated to the feature.

Reply:
intentional, I noticed an unused import